src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp

```diff
@@ -327,6 +338,79 @@ void PeHeuristics::getVisualBasicHeuristics()
 	}
 }
 
+int32_t PeHeuristics::get_int32_unaligned(const uint8_t * codePtr)
+{
+	const int32_t * codePtrInt32 = reinterpret_cast<const int32_t *>(codePtr);
+
+    return codePtrInt32[0];
+}
+
+/**
+ * Parses the code, follows NOPs or JMPs
+ */
+const uint8_t * PeHeuristics::skip_NOP_JMP8_JMP32(const uint8_t * codeBegin, const uint8_t * codePtr, const uint8_t * codeEnd, size_t maxCount)
```

Comment:
In C++, please qualify types such as `size_t` with `std::` ([reasons](https://cs-blog.petrzemek.net/2015-07-18-zajimavost-z-cpp-size_t-vs-std-size_t)).